src/Microsoft.DotNet.Build.Tasks/VersionTools/LocalUpdatePublishedVersions.cs

```diff
@@ -5,31 +5,61 @@
 using Microsoft.Build.Framework;
 using Microsoft.DotNet.VersionTools.Automation;
 using System.Diagnostics;
-using System.Linq;
 
 namespace Microsoft.DotNet.Build.Tasks.VersionTools
 {
     public class LocalUpdatePublishedVersions : BuildTask
     {
         [Required]
-        public ITaskItem[] ShippedNuGetPackage { get; set; }
+        public string[] ShippedNuGetPackage { get; set; }
```

Comment:
Why the switch to string[]? This potentially break users of this task.

Reply:
Maybe there's some case I'm not aware of, but in my experience this doesn't break anyone using the task through msbuild. The metadata is just ignored, which is fine since this task doesn't use it. I think I did this to avoid a Select, but it's not worth changing--I'll revert.